conf/site.conf

```diff
@@ -8,8 +8,8 @@ CONF_VERSION = "1"
 # Modify the following if you need to change the types of images generated
 # most of the time, this is set by the machine conf or in a custom image
 # recipe.
-IMAGE_FSTYPES += "ext4.gz tar.xz wic.xz wic.bmap"
-IMAGE_FSTYPES_remove_qemuall = "wic.xz wic.bmap"
+IMAGE_FSTYPES += "wic.xz wic.bmap"
+IMAGE_FSTYPES_remove_qemuall = "tar.bz2 tar.xz tar tar.gz wic.xz wic.bmap"
```

Comment:
might save some time on builds?

Reply:
that is right,  currently for yoe-sdk-image this takes 5+ mins for each type.